src/migration/utilities.js

```diff
@@ -1,14 +1,32 @@
-function getDependency(artifacts, network, deployer, path) {
+function initializeContract(path, deployer, accounts) {
+  const contract = require("truffle-contract")
+
+  const Contract = contract(require(path))
+  Contract.setProvider(deployer.provider)
+  Contract.setNetwork(deployer.network_id)
+  // For some reason the automatic value calculation is not working, hence we do:
+  Contract.defaults({
+    from: accounts[0],
+    gas: 8e6,
```

Comment:
Is this for deployment? We need to be a bit careful as this exceeds the default ganache limit. Would 6M be sufficient for deployment?

Reply:
Yeah, it is for deployment. I can change it for 6.5M gas. 
Generally, I am very unsatisfied that we don't get these numbers automatically if we migrate BatchExchange as a dependency